recipe/libffi.pc

```diff
@@ -0,0 +1,11 @@
+prefix=/c/ci/libffi_1601349976473/_h_env/Library
```

Comment:
hmm, this hard-coded _h_env (%PREFIX%) path looks actually bad.  Shouldn't. we. instead write here @PREFIX@, and then using sed to modify it to point to the correct - current - PREFIX directory?

Reply:
maybe even better to spend to libffi the pc file itself?

Reply:
Yes, it needs to be set to PREFIX after copying, so that [conda will automatically fix it during installation](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#text-files-with-prefix-files). But I wonder why that wouldn't work out-of-the-box with the pc file generated during build.

Reply:
This is actually fine. On windows, this prefix is overridden by default to two directories up.
Here I added the pc file because it is entirely missing for the conda package.

zlib on the other hand has an absolute path set in field libs.private, which is a big no. That's why I use sed for this one.

Reply:
I stumbled on the same ffi issue for other packages. So I am fixing it the right way instead:
https://github.com/AnacondaRecipes/libffi-feedstock/pull/10